lib/src/client.dart

```diff
@@ -99,14 +99,14 @@ class Client extends Object with event.Emitter {
     return true;
   }
 
-  _didClose([int code = 0, String reason = "", bool force = false]) {
+  void _didClose([int code = 0, String reason = "", bool force = false]) {
     if (readyState != CONNECTING &&
         readyState != OPEN &&
         readyState != CLOSING) {
             throw 'INVALID_STATE_ERR';
     }
     if (_ir != null) {
-        _ir.nuke();
+//        _ir.nuke();
```

Comment:
Is this supposed to be commented out?

Reply:
yes... this `nuke()` method doesn't actually exist, so I guess this code path was just never hit in actual usage